cluster-autoscaler/config/autoscaling_options.go

```diff
@@ -295,6 +295,8 @@ type AutoscalingOptions struct {
 	ProvisioningRequestEnabled bool
 	// AsyncNodeGroupsEnabled tells if CA creates/deletes node groups asynchronously.
 	AsyncNodeGroupsEnabled bool
+	//LastProvisioningRequestSeenTime is the time of the last provisioning request was attempted to be processed
+	LastProvisioningRequestSeenTime *time.Time
```

Comment:
As discussed offline, I think a name referring to the processing time (vs observing time) would be more appropriate.

But a higher level comment, AutoscalingOptions is not a good place to store any kind of state. My suggestion would be to:
1. Add a LastProvReqTime getter directly to the provreq injector
2. Pass the injector directly to the LoopTrigger

Reply:
Got it. Updated accordingly.